cmd/kubeadm/app/cmd/reset.go

```diff
@@ -295,3 +311,19 @@ func resetConfigDir(configPathDir, pkiPathDir string) {
 		}
 	}
 }
+
+//Checks if is control plane
+func isControlPlane(cfg *kubeadmapi.InitConfiguration) bool {
+
+	if cfg == nil {
+		return false
+	}
+
+	filepath := kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.KubeAPIServer, kubeadmconstants.GetStaticPodDirectory())
```

Comment:
Comment here stating that only the control plane has a static pod directory and that is how it is being determined if this is the control plane could be nice.